pyccel/ast/builtins.py

```diff
@@ -208,12 +205,11 @@ class PythonBool(PyccelInternalFunction):
     """
     __slots__ = ()
     name = 'bool'
-    _dtype = NativeBool()
-    _precision = -1
+    _static_type = PythonNativeBool()
```

Comment:
what's the use of `_static_type`?

Reply:
I have added an explanation in the docs: c4672465